com.unity.shadergraph/Editor/Importers/ShaderGraphImporter.cs

```diff
@@ -14,7 +14,16 @@
 namespace UnityEditor.ShaderGraph
 {
     [ExcludeFromPreset]
+#if ENABLE_HYBRID_RENDERER_V2 && UNITY_2020_1_OR_NEWER
+    // Bump the version number when Hybrid Renderer V2 is enabled, to make
```

Comment:
hmm... I wondering if we should do it for all version, not only hybrid (given we are doing shader target anyway). Is there any benefit to only do it for hybrid? Also  this PR only affect SG since refactor right? so it will always affect UNITY_2020_1_OR_NEWER. no need for it.

Reply:
The reasoning here is that we have used `UNITY_2020_1_OR_NEWER` as part of the ifdef guard everywhere (because V2 doesn't work on older versions), so we just wanted it to be the same here.

Another, and in my opinion maybe a better way would be to just use the one define, and have code in one file that produces an error if the V2 define is enabled on older versions.

Reply:
I discussed this a bit, and many people agree with removing the Unity version ifdef, so I think we will make a separate PR to do it.

Reply:
Yes. Please use ENABLE_HYBRID_RENDERER_V2 define only. Let's add error in the DOTS code if this define is enabled and too old Unity version is used.

Reply:
The ifdef has now been changed.